habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hierarchical_policy/defined_skills/oracle_skills_nogrip.yaml

```diff
@@ -0,0 +1,67 @@
+# Oracle skills that will teleport to the skill post-condition. When automatically setting predicates you may want to run the simulation in kinematic mode:
```

Comment:
Can you add an explanation to what ignore_grip changed here ?

Reply:
Added a comment in default_structured_config.py